### PR TITLE
Big riot/redux/routing refactor

### PR DESCRIPTION
**Pull request by PeteCoward**
WIP

Needs a lot of tidy up and chasing down how to access the manifest sensibly

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+    } finally {
+        // swallow this error
+    }
+}
+
+if (
+    !window.certChain
+) {
+    InitialiseCertChain();
+}
```
Comment (PeteCoward) [inline src/app.js:32]: intention is for this to be a big list of `initialiszeThis` function calls

File: src/riot/Screens/Login.riot.html [hunk 2]
```diff
@@ -71,6 +71,7 @@ <h2 class="header-no-image">{ TRANSLATIONS.loggingAsGuest() }</h2>
             },
 
             onBeforeMount(props, state) {
+
```
Comment (md8n) [inline src/riot/Screens/Login.riot.html:74]: Just delete this empty line - and then it's one less file in this :pr:

File: src/riot/App.riot.html [hunk 3]
```diff
@@ -99,132 +129,6 @@
...
-            if (!!wagtailPage.meta) {
-                // Existing page format
-                const type = wagtailPage.meta.type;
-                return type.substring(type.lastIndexOf(".") + 1);
-            }
-            // New manifest format
-            const type = wagtailPage.loc_hash;
-            return type.substring(type.lastIndexOf("/") + 1);
-        }
+        export default App
```
Comment (md8n) [inline src/riot/App.riot.html:132]: Really nice work - thanks

**Review by md8n — APPROVED** [review]

File: src/riot/Screens/Login.riot.html (hunk 2 above)
Comment (PeteCoward) [reply src/riot/Screens/Login.riot.html:74]: whoops!

File: src/riot/App.riot.html [hunk 4]
```diff
@@ -99,132 +129,6 @@
...
-                const page = await getPage();
-                const pageModel = getWagtailPageModelName(page);
-
-                if (this.areCompletionsReady() && !window.certChain) {
-                    InitialiseCertChain();
-                }
-
-                this.update({
-                    wagtailPage: page,
-                    contentType: pageModel,
```
Comment (md8n) [inline src/riot/App.riot.html:186]: Setting this value `contentType` into the `state` is absolutely necessary for the `Sync` page to actually show up.  This is the output from the call to `getWagtailPageModelName` above

**Review by md8n — CHANGES_REQUESTED** [review]

**Review by md8n — APPROVED** [review]

File: src/riot/Screens/Login.riot.html (hunk 2 above)
Comment (PeteCoward) [inline src/riot/Screens/Login.riot.html:74]:
```suggestion
```

**Comment by md8n** [comment]
OK - actually approved now - feel free to merge in

**Pull request merged**